### ISSUE TITLE
Create a book view

### ISSUE THREAD

**Issue by esulu** [esulu/COMP3005-Project#6]
When a book is selected, the following information can be viewed:
- author(s)
- genre
- publisher
- page count 
- price
- book cover
- year

The user will also be given the option to add as many books as they like to their cart (limited by how much is available)  -> this will be done in another ticket

Depends on #7
